dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/ProgramMessageController.java

```diff
@@ -89,9 +89,15 @@ public class ProgramMessageController extends AbstractFullReadOnlyController<Pro
   @ResponseBody
   public List<ProgramMessage> getProgramMessages(
       @RequestParam(required = false) Set<String> ou,
-      @Deprecated(since = "2.41") @RequestParam(required = false) UID programInstance,
+      @OpenApi.Param(value = Enrollment.class)
```

Comment:
For all 4 `UID` where you added the `@OpenAPI.Param` annotation in the last commit it lacks the `UID.class` in the `value`, e.g. `@OpenApi.Param(value = { UID.class, Enrollment.class})` not `@OpenApi.Param(value = Enrollment.class)`.